tests/examples/awful/template.lua

```diff
@@ -154,6 +159,24 @@ local function client_widget(c, col, label)
 end
 
 -- Add all wiboxes
+
+-- Fix the wibox geometries that have a dependency on their content
+for _, d in ipairs(drawin.get()) do
+    local w = d.get_wibox and d:get_wibox() or nil
+    if w then
+        -- Force a full layout first as widgets with as the awful.popup have
+        -- interdependencies between the content and the container
```

Comment:
I still fail to parse this comment as English.
- "Force a full layout first": Okay, so before anything else is done, something(?) is (re-?)layouted first.
- "as widgets": Because of widgets
- "with as the": Sorry, I'm lost

This is the first commit that I look at and just the second hunk in the diff...